powertools-parameters/pom.xml

```diff
@@ -73,6 +73,10 @@
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>url-connection-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
```

Comment:
I have taken this as a dep on the powertools-parameters project. I considered refactoring into submodules so you don't take the dynamodb dependency if you use parameters at all, but agree with @jeromevdl that this is better for a v2 change, as we'll have to move the existing providers down too and this will break.

Reply:
yep, it will burden the jar, we'll split it in v2